packages/core/src/servient.ts

```diff
@@ -160,6 +160,18 @@ export default class Servient {
         }
     }
 
+    public removeThing(thingId: string): boolean {
+        if (this.things.has(thingId)) {
+            this.servers.forEach( (server) => {
+                server.destroy(thingId);
```

Comment:
`server.destroy` returns a promise, we should wait until competition otherwise we might create problems down the road

Reply:
+1

Reply:
Question: `removeThing()` in servient.ts is a sync call while `server.destroy()` is async call.
What is the best way to handle this or to put it differently how to wait for all promises to be returned. Any proposal @relu91

Reply:
What I have now obviously doesn't work. The return does not return the outer call.

https://github.com/eclipse/thingweb.node-wot/blob/36029d479ab50279115cfb4a680f01f65c871654/packages/core/src/servient.ts#L167-L174

Reply:
Question: Should the servient.removeThing() return a promise?
Note: servient.addThing() does return a plain boolean

Reply:
I renamed it to `destroyThing()` and returning a promise also.
Moreover, this is a function called in core only.. so we _could_ still change it in future...